test/e2e_node/e2e_service.go

```diff
@@ -185,6 +179,18 @@ func (es *e2eService) startServer(cmd *healthCheckCommand) error {
 		cmd.Cmd.Stdout = outfile
 		cmd.Cmd.Stderr = outfile
 
+		// Killing the sudo command should kill the server as well.
+		attrs := &syscall.SysProcAttr{}
+		// Hack to set linux-only field without build tags.
```

Comment:
We should ideally not cross compile these tests. They work only on linux as of now.